data/events/2025/wollongong/main.yml

```
@@ -0,0 +1,144 @@
+name: "2025-wollongong" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
+year: "2025" # The year of the event. Make sure it is in quotes.
+city: "Wollongong" # The displayed city name of the event. Capitalize it.
```

If you want to add Wollongong to the [previous Australian events](https://github.com/search?q=repo%3Adevopsdays%2Fdevopsdays-web%20event_group%3A%20%22Down%20Under%22&type=code), you can add the following: 
```suggestion
city: "Wollongong" # The displayed city name of the event. Capitalize it.
event_group: "Down Under"
```

---

Thanks done 🙏🏻